requirements.txt

```diff
@@ -0,0 +1,4 @@
+Django>=4.0.0,<4.1.0
```

Comment:
Normally in Python in `requirements.txt` file you pin the packages to an exact version, rather than a range of versions. This ensures repeatable builds among different environments.

Read https://nvie.com/posts/pin-your-packages/ to learn why.